### PR TITLE
Add new stats to monitoring API docs

### PR DESCRIPTION
**Pull request by dedemorton**
Summary of changes:
- Added new stats for persistent queues and cgroups
- Updated JSON examples with new data based on latest build (not always necessary, but ensures that I don't miss things that have changed)
- Added links to subsections to make it easier for users to navigate
- Made sure that all topics had links to common options in a consistent location

### CODE REVIEW

**Comment by dedemorton** [comment]
This resolves the API changes described in #6501

**Comment by talevy** [comment]
pq stuff looks good, can't comment on the rest

File: docs/static/monitoring-apis.asciidoc [hunk 1]
```diff
@@ -399,12 +433,87 @@ Example response:
...
+--------------------------------------------------
+
+[[os-stats]]
+==== OS Stats
+
+When Logstash is running in a container, the following request returns a JSON document that
+contains cgroup information to give you a more accurate view of CPU load, including whether
+the container is being throttled. 
+
+//REVIEWERS: I couldn't easily test this because I'm not running Logstash in a Linux container. The example is based on info in the PR. Can someone confirm?
```
Comment (suyograo) [inline docs/static/monitoring-apis.asciidoc]: Here's a better example, from a linux box with cgroups configured

```
"cgroup" : {
                "cpuacct" : {
                  "control_group" : "/elastic1",
                  "usage_nanos" : 378477588075
                },
                "cpu" : {
                  "control_group" : "/elastic1",
                  "cfs_period_micros" : 1000000,
                  "cfs_quota_micros" : 800000,
                  "stat" : {
                    "number_of_elapsed_periods" : 4157,
                    "number_of_times_throttled" : 460,
                    "time_throttled_nanos" : 581617440755
                  }
                }
              }
```

**Comment by suyograo** [comment]
LGTM

**Comment by dedemorton** [comment]
@suyograo I'm going to push my changes and commit them, but I wanted to point out that we report reloads in two different places. Not sure if this is intentional or a bug.

**Pull request closed without merging**